tests/test_evaluations/test_evaluation_functions.py

```diff
@@ -117,6 +117,31 @@ def test_evaluation_list_per_fold(self):
             self.assertIsNotNone(evaluations[run_id].value)
             self.assertIsNone(evaluations[run_id].values)
 
+    def test_evaluation_list_sort(self):
+        openml.config.server = self.test_server
+        size = 10
+        task_id = 1769
+        # Get all evaluations of the task
+        unsorted_eval = openml.evaluations.list_evaluations(
+            "predictive_accuracy", offset=0, task=[task_id])
+        # Get top 10 evaluations of the same task
+        sorted_eval = openml.evaluations.list_evaluations(
+            "predictive_accuracy", size=size, offset=0, task=[task_id], sort_order="desc")
+
```

Comment:
Please add a sanity check that ensures no empty dictionaries are returned. As is, it seems like the current test only passes because no results are returned.